### PR TITLE
fixes: to migration docs and kured

### PR DESCRIPTION
**Pull request by crssnd**
**What this PR does / why we need it**: to fix some issues with the migration docs and kured

### CODE REVIEW

File: config/config/common-config.yaml [hunk 1]
```diff
@@ -579,6 +579,7 @@ kured:
     slack:
       enabled: false
       channel: ""
+      botToken: ""
```
Comment (robinAwallace) [inline config/config/common-config.yaml:582]: This is part of secret and not common-config.

**Review by robinAwallace — CHANGES_REQUESTED** [review]

File: config/secrets/sc-secrets.yaml [hunk 2]
```diff
@@ -58,9 +58,9 @@ alerts:
     apiUrl: somelongsecret
   opsGenie:
     apiKey: somelongsecret
-# kured:
-#   slack:
-#     botToken: "set-me"
+#kured:
+#  slack:
+#    botToken: "me"
```
Comment (robinAwallace) [inline config/secrets/sc-secrets.yaml]:
```suggestion
#    botToken: "set-me"
```

File: helmfile/50-applications.yaml [hunk 3]
```diff
@@ -45,7 +45,7 @@ releases:
   chart: ./charts/kured-secret
   version: 0.1.0
   missingFileHandler: Error
-  installed: {{ .Values.kured.enabled }}
+  installed: {{ .Values.kured.notification.slack.enabled }}
```
Comment (robinAwallace) [inline helmfile/50-applications.yaml:48]:
```suggestion
  installed: {{ and .Values.kured.enabled .Values.kured.notification.slack.enabled }}
```

**Review by robinAwallace — APPROVED** [review]

**Review by davidumea — APPROVED** [review]

File: migration/v0.25.x-v0.26.x/upgrade-apps.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Upgrade v0.24.x to v0.25.x
+# Upgrade v0.25.x to v0.26.x
```
[the remark this answers is not in the document]
Comment (crssnd) [reply migration/v0.25.x-v0.26.x/upgrade-apps.md:1]: I ended up adding almost all the instructions :)

File: migration/v0.25.x-v0.26.x/upgrade-apps.md [hunk 5]
```diff
@@ -34,14 +34,30 @@
...
+    kured:
+      slack:
+        botToken: abcxyz-...
+    ```
+    b. Enable the slack notification and add the channel in `${CK8S_CONFIG_PATH}/common-config.yaml`:
+    ```
+    notification:
+      slack:
+        enabled: true
+        channel: notification-channel
```
Comment (viktor-f) [inline migration/v0.25.x-v0.26.x/upgrade-apps.md:58]:
```suggestion
        channel: kured-notification
```

**Review by viktor-f — APPROVED** [review]

**Pull request merged**